docs/markdown/recursor/settings.md

```diff
@@ -511,7 +511,7 @@ to detect and act on infected hosts.
 Protobuf export to a server is enabled using the `protobufServer()` directive:
 
 ```
-protobufServer("192.0.2.1:4242" [[[[[, timeout], maxQueuedEntries], reconnectWaitTime], maskV4], maskV6])
+protobufServer("192.0.2.1:4242" [[[[[[, timeout], maxQueuedEntries], reconnectWaitTime], maskV4], maskV6], taggedOnly])
```

Comment:
I wonder if there is a more readable way to write 'these ordered arguments are all optional'. Alternatively we could make them named of course.